prowler/providers/azure/services/sqlserver/sqlserver_service.py

```diff
@@ -44,6 +45,11 @@ def __get_sql_servers__(self):
                     vulnerability_assessment = self.__get_vulnerability_assesments__(
```

Comment:
Please, add separated functions for `list_by_server`?

Reply:
Good idea! Done